BuildResidentialHPXML/measure.rb

```diff
@@ -3970,14 +4039,55 @@ def self.set_heating_systems(hpxml, runner, args)
       heating_efficiency_percent = args[:heating_system_heating_efficiency_percent]
     end
 
+    fraction_heat_load_served = args[:heating_system_fraction_heat_load_served]
+    fraction_heat_load_served_2 = args[:heating_system_fraction_heat_load_served_2]
+    fraction_heat_load_served -= fraction_heat_load_served_2
```

Comment:
I'm totally confused. Why are you subtracting heat_load_served_2 from heat_load_served?

Reply:
For example: if the user left the primary at 100% and then entered a secondary system at 40%, this ensures that the total load fraction is 1.0.

Reply:
You are thinking of EnergyPlus's sequential load fractions, which are defined differently than HPXML's fractions load served. The user should enter 60% and 40% for that situation and they should be directly translated to the HPXML file.

Reply:
So should an error be thrown if the two fractions don't add up to 1.0?

Reply:
An error will be issued if they try to use the HPXML file in the HPXMLtoOpenStudio measure. So... I guess not?